Formula/gedit.rb

```diff
@@ -97,7 +96,7 @@ def post_install
       -I#{gtksourceview4.opt_include}/gtksourceview-4
       -I#{gtkx3.opt_include}/gtk-3.0
       -I#{harfbuzz.opt_include}/harfbuzz
-      -I#{include}/gedit-3.36
+      -I#{include}/gedit-3.38
```

Comment:
Can we do `#{version.major}.#{version.minor}` here

Reply:
https://github.com/Homebrew/homebrew-core/pull/60964#discussion_r487119340